### DISCUSSION TITLE
Prefixed defines

### DISCUSSION THREAD

**Discussion by SolitudeSF** [HapticX/happyx#46] [General]
Consider using `happyx` prefixed defines, for backend selection. Non-prefixed defines are fine for applications, but library code shouldn't be so eager to use unprefixed and generic names as `micro`.

**Comment by Ethosa** [comment]
mean you that all function names should starts with `happyx`?
as ex. `answer` -> `happyxAnswer`?

**Reply by SolitudeSF** [reply]
> Multiple options for HTTP server! Happyx use asynchttpserver as default HTTP server (httpx via -d:httpx and microhttpserver via -d:micro as alternative HTTP servers).

httpx -> happyxHttpx, micro -> happyxMicro

**Reply by Ethosa** [reply]
sounds logical. I agree with you

**Comment by Ethosa** [comment]
Available since `1.0.0`
